python-api/tests/livy_test_utils.py

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class LivyTestUtils(object):
+
+    @staticmethod
+    def simple_spark_job(context):
+        sc = context.sc
+        elements = []
+        elements.__add__(10)
```

Comment:
It's really weird to call these methods directly. Whenever you do that, it generally means you're doing something wrong.

This should be `elements.append(10)`. Or you could just say `elements = [ 10, 20, 30 ]`.